CHANGES.txt

```diff
@@ -27,6 +34,28 @@ Backwards Incompatibilities
 
 - Drop support for Python 2.7, 3.4, 3.5, 3.6, and 3.7.
 
+- Remove ``AcceptValidHeader``, ``AcceptNoHeader`` and ``AcceptInvalidHeader``.
+  These classes are consolidated into ``Accept`` with a ``header_state``
+  attribute for users that need to know the exact state of the header.
+  See https://github.com/Pylons/webob/pull/460
+
+- Remove previously-deprecated ``webob.acceptparse.MIMEAccept``.
+  See https://github.com/Pylons/webob/pull/460
+
+- Remove previously-deprecated ``webob.acceptparse.Accept.__iter__``.
+  See https://github.com/Pylons/webob/pull/460
+
+- ``webob.acceptparse.Accept`` methods, ``best_match``, ``quality``, and
+  ``__contains__`` are now thin wrappers around ``acceptable_offers`` which
+  modifies their behavior slightly:
+
+  - Offers containing wildcards are no longer allowed.
+  - A tuple can no longer be an offer containing server-side quality values.
+  - An offer will only match a wildcard clause in the header, such as ``*/*``
+    or ``text/*`` if it does not match something more specific.
```

Comment:
If I remember correctly, their behaviour will be modified more than slightly: the switch from the old algorithms to the one in `.acceptable_offers` involved many changes - some of which were documented in several of the docstrings that are being removed, but even those docstrings didn't include everything. As these methods have been deprecated for such a long time, it may be more helpful and accurate/precise to say that the methods have switched to using the algorithm in `.acceptable_offers` to conform to RFC 7231?

(I also had trouble understanding "A tuple can no longer be an offer containing server-side quality values." - is it saying that offers can no longer contain server-side quality values?)

(The same text is used in the docstrings of `.best_match`, `.quality` and `.__contains__`, so this comment also applies to them.)